defaults/main.yml

```diff
@@ -22,5 +22,20 @@ gitlab_ldap_bind_dn: "CN=Username,CN=Users,DC=example,DC=com"
 gitlab_ldap_password: "password"
 gitlab_ldap_base: "DC=example,DC=com"
 
+# Timezone
+gitlab_time_zone: "CET"
+
+# Gitlab mail
+gitlab_email_from: 'gitlab@gitlab.com'
+gitlab_email_display_name: 'Gitlab'
+gitlab_email_reply_to: 'gitlab@gitlab.com'
+
 # Probably best to leave this as the default, unless doing testing.
 gitlab_restart_handler_failed_when: 'gitlab_restart.rc != 0'
+
+# CI config
+gitlab_ci_external_url: "http://ci.gitlab.com"
```

Comment:
Since many people (maybe most?) won't have any integration with CI configured, I'd rather leave these as empty/blank defaults, and that way the configuration won't be added unless there are values. Otherwise maybe add a variable that triggers adding the rest or not, titled `gitlab_ci_enable_configuration` or something like that.